packages/editor/src/components/post-author/index.js

```diff
@@ -1,49 +1,165 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __, _n } from '@wordpress/i18n';
 import { withInstanceId, compose } from '@wordpress/compose';
 import { Component } from '@wordpress/element';
 import { withSelect, withDispatch } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
 
 /**
  * Internal dependencies
  */
 import PostAuthorCheck from './check';
 
+/**
+ * External dependencies
+ */
+import Autocomplete from 'accessible-autocomplete/react';
+
+import { debounce } from 'lodash';
+
 export class PostAuthor extends Component {
 	constructor() {
 		super( ...arguments );
 
 		this.setAuthorId = this.setAuthorId.bind( this );
+		this.suggestAuthor = this.suggestAuthor.bind( this );
+		this.getCurrentAuthor = this.getCurrentAuthor.bind( this );
+		this.resolveResults = this.resolveResults.bind( this );
```

Comment:
I'd encourage some sense of organization to these assignments. For me, roughly:

- super call
- instance function bindings (`suggestAuthor`, etc)
- other instance assignments (`searchCache`)
- state assignment (technically similar to previous, but special in its own way)

Reply:
> I'd encourage some sense of organization to these assignments

I'm not sure what you are suggesting, additional inline comments or better naming?

Reply:
It's a suggestion of the ordering. `this.requestResults` is a function binding, so should probably occur among this set, i.e. after `this.resolveResults` and before the `this.state` assignment.

Reply:
thanks for clarifying; addressed in https://github.com/WordPress/gutenberg/pull/7385/commits/624c49d907b2e6f5d1b6937c7dd56f338b6265fc